exercises/concept/developer-privileges/DeveloperPrivilegesTests.cs

```diff
@@ -30,23 +30,54 @@ public void GetAdmin()
 
     [Fact]
     [Task(2)]
-    public void GetDevelopers()
+    public void GetDevelopers_Bertrand()
     {
         var authenticator = new Authenticator();
         var developers = authenticator.Developers;
         string[] actual =
         {
             developers["Bertrand"]?.Email,
             developers["Bertrand"]?.FacialFeatures.EyeColor,
-            developers["Anders"]?.FacialFeatures.PhiltrumWidth.ToString(CultureInfo.InvariantCulture),
-            developers["Anders"]?.NameAndAddress[1]
+            developers["Bertrand"]?.FacialFeatures.PhiltrumWidth.ToString(CultureInfo.InvariantCulture),
+            developers["Bertrand"]?.NameAndAddress[0],
+            developers["Bertrand"]?.NameAndAddress[1],
+            developers["Bertrand"]?.NameAndAddress[2]
         };
         string[] expected =
         {
             "bert@ex.ism",
             "blue",
+            "0.8",
+            "Bertrand",
+            "Paris",
+            "France"
+        };
+        Assert.Equal(expected, actual);
+    }
+
+    [Fact]
+    [Task(3)]
+    public void GetDevelopers_Anders()
+    {
+        var authenticator = new Authenticator();
+        var developers = authenticator.Developers;
+        string[] actual =
+        {
+            developers["Anders"]?.Email,
+            developers["Anders"]?.FacialFeatures.EyeColor,
+            developers["Anders"]?.FacialFeatures.PhiltrumWidth.ToString(CultureInfo.InvariantCulture),
+            developers["Anders"]?.NameAndAddress[0],
+            developers["Anders"]?.NameAndAddress[1],
+            developers["Anders"]?.NameAndAddress[2]
+        };
+        string[] expected =
+        {
+            "anders@ex.ism",
+            "brown",
             "0.85",
-            "Redmond"
+            "Anders",
+            "Redmond",
+            "USA"
```

Comment:
Great work! I'll approve and merge this PR. If you're interested in doing another PR, you could maybe make the checks be individual asserts and not using an array (our fault, not yours)? Like this:

```csharp
Assert.NotNull(developers["Anders"]);
Assert.Equal("anders@ex.ism", developers["Anders"].Email)
Assert.Equal("brown", developers["Anders"].EyeColor)
Assert.Equal(0.85, developers["Anders"]?.FacialFeatures.PhiltrumWidth, precision: 2),
Assert.Equal("Anders", developers["Anders"].NameAndAddress[0])
Assert.Equal("Redmond", developers["Anders"].NameAndAddress[1])
Assert.Equal("USA" developers["Anders"].NameAndAddress[2])
```